Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -21,6 +21,13 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
     /// `ChatMessageController` instance used to obtain the message data.
     public var messageController: ChatMessageController!
 
+    /// The channel which the actions will be performed.
+    public var channel: ChatChannel? {
+        didSet {
+            channelConfig = channel?.config
+        }
+    }
+
     /// `ChannelConfig` that contains the feature flags of the channel.
     public var channelConfig: ChannelConfig!
```

Comment:
Is this still used anywhere?

Reply:
however, this is public and mutable. Customers might be mislead by changing things here and expecting the change to be reflected for all its properties.

Reply:
They will be changed. Everything here is backwards compatible 👍